base/abstractarray.jl

```diff
@@ -2009,6 +2009,24 @@ function (==)(A::AbstractArray, B::AbstractArray)
     return anymissing ? missing : true
 end
 
+"""
+    <(A::AbstractVector, B::AbstractVector)
+
+Returns true when `A` is less than `B` in lexicographic order using <
+to perform comparisons of the elements of both vectors.
```

Comment:
"Returns true" is a bit misleading since it can return `missing` like `<`.
```suggestion
Compare `A` and `B` in lexicographic order by applying `<` to their elements.
```